### PR TITLE
add INSTANCE_NAME and INSTANCE_DESCRIPTION settings

### PR DESCRIPTION
**Pull request by ratsclub**

### CODE REVIEW

File: main/templates/main/layout.html [hunk 1]
```diff
@@ -7,7 +7,8 @@
         <title>{% block title %}{% endblock %}</title>
 
         {% if not request.subdomain %}
-        <meta name="description" content="Blogging platform for minimalists.">
+        {# TODO: how to add it here? #}
```
Comment (ratsclub) [inline main/templates/main/layout.html]: This is the only place I couldn't figure how to add the description... Would you be able to help me here, please?

Comment (theo429) [reply main/templates/main/layout.html:10]: Sure! Add it on the request somewhere here: https://github.com/mataroa-blog/mataroa/blob/fbb6ef628935af95a03a70a28e176e588418c2da/main/middleware.py#L39

Comment (sirodoht) [reply main/templates/main/layout.html]: Sure! Add it on the request somewhere here: https://github.com/mataroa-blog/mataroa/blob/fbb6ef628935af95a03a70a28e176e588418c2da/main/middleware.py#L39

**Comment by ratsclub** [comment]
@sirodoht If you find a better name than `INSTANCE_*`, I can change it too.

File: main/views/general.py [hunk 2]
```diff
@@ -98,7 +98,14 @@ def index(request):
     if request.user.is_authenticated:
         return redirect("blog_index")
 
-    return render(request, "main/landing.html")
+    return render(
+        request,
+        "main/landing.html",
+        {
+            "instance_name": settings.INSTANCE_NAME,
+            "instance_description": settings.INSTANCE_DESCRIPTION
```
Comment (sirodoht) [inline main/views/general.py:106]: let's remove it from here, since we include it in the request object

File: main/templates/main/landing.html [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
+{% block title %}{{ instance_name }} - {{ instance_description }}{% endblock %}
 
 {% block content %}
 <main>
-    <h1>Mataroa blog</h1>
-    <p>
-        Naked blogging platform, for minimalists. <em>Just write.</em>
-    </p>
+    <h1>{{ instance_name }}</h1>
+    <p>{{ instance_description }} <em>Just write.</em></p>
```
Comment (sirodoht) [inline main/templates/main/landing.html:8]: this is kinda annoying. having the `<em>Just write.</em>` there :| not sure how to improve though

File: main/views/general.py (hunk 2 above)
Comment (sirodoht) [reply main/views/general.py:106]: actually maybe add the `settings.INSTANCE_NAME` also on the request.

**Pull request closed without merging**